samples/bluetooth/mesh/light_ctrl/src/model_handler.c

```diff
@@ -80,7 +85,7 @@ static void start_new_light_trans(const struct bt_mesh_lightness_set *set,
 				  struct lightness_ctx *ctx)
 {
 	uint32_t step_cnt = abs(set->lvl - ctx->current_lvl) / PWM_SIZE_STEP;
-	uint32_t time = set->transition ? set->transition->time : 0;
+	uint32_t time =	 set->transition ? set->transition->time : 0;
```

Comment:
Remove redundant space
```suggestion
	uint32_t time = set->transition ? set->transition->time : 0;
```